src/firmware_new/boards/frankie_v1/freertos.c

```diff
@@ -276,7 +277,7 @@ void initIoNetworking()
 {
     // TODO channel and robot_id need to be hooked up to the dials on the robot, when
     // available https://github.com/UBC-Thunderbots/Software/issues/1517
-    unsigned channel = 0;
+    unsigned short int channel = 0;
```

Comment:
Is the purpose of using the `short` keyword to save memory?

Reply:
yeah it can, but in this case I'm just trying to make the values consistent with function arguments